demos/demo_2d_north_sea.py

```diff
@@ -110,6 +108,17 @@ def read_station_data():
 dt = 3600.0
 t_export = 3600.0
 
+# .. raw:: html
+#
+#    <!-- in the CI we only run a few timesteps; commented out in html for the web version
+
+if os.environ['THETIS_REGRESSION_TEST']:
```

Comment:
```suggestion
if os.environ.get('THETIS_REGRESSION_TEST'):
```

Otherwise this will throw an error locally as the flag isn't set outside of testing. In the [3D channel demo](https://github.com/thetisproject/thetis/blob/master/demos/demo_3d_channel.py#L19) we should also add the flag to alter the end time as per the [example](https://github.com/thetisproject/thetis/blob/master/examples/channel3d/channel3d.py#L26) (100s test without it). Will also need to add a comment/comment out test in html version.

I'm running other stuff locally but indicatively: single tracer and 2d channel 8-10s, 2d channel with boundaries 30s, multiple tracers demo takes 20s. They're probably OK without flags.